test/unit/test_ArithmeticExpressions.cpp

```diff
@@ -46,7 +46,7 @@ TEST_F(ArithmeticExpressions_test, test_One_Arg_Multiplication)
     PTRef two = logic.mkConst("2");
     PTRef one = logic.mkConst("1");
     PTRef a = logic.mkNumVar("a");
-    PTRef div = logic.mkNumDiv(a, two);
+    PTRef div = logic.mkRealDiv(a, two);
```

Comment:
We could maybe have unit tests for `LALogic::mkNumDiv` for both cases, where the instance is actually `LRALogic` and when it is `LIALogic`.